tests/dataSources_test.go

```diff
@@ -803,15 +803,41 @@ var _ = Describe("DataSources", func() {
 				waitUntilDeployed()
 			})
 
-			It("[test_id:7469] should create DataImportCron", func() {
-				Expect(apiClient.Get(ctx, dataImportCron.GetKey(), dataImportCron.NewResource())).To(Succeed(), "custom DataImportCron not created")
+			It("[test_id:7469] should create DataImportCron in golden images namespace", func() {
+				Expect(apiClient.Get(ctx, dataImportCron.GetKey(), dataImportCron.NewResource())).To(Succeed(), "custom DataImportCron created")
+			})
+
+			It("[test_id:????] should create DataImportCron in other namespace", func() {
```

Comment:
Can you create a context and put all the tests in different namespace there?
Then you can use `BeforeEach` and `AfterEach` to create and delete the temporary namespace.